scripts/install_via_conda.sh

```diff
@@ -34,7 +34,7 @@ else
 fi
 
 # install other deps
-conda install -y numpy sphinx pytest flake8 ipywidgets ipython
+conda install -y numpy sphinx pytest flake8 ipywidgets ipython scikit-learn
```

Comment:
This only affects CircleCI. We could completely remove sklearn dependencies from tests, but it may be good to test end-to-end if possible to catch any changes which might break our methods when using sklearn, since it may be a common use case for the interpretable models. Once we have PyTorch linear models, we can test both, and just skip the sklearn tests if not available?